apps/website-new/docs/en/guide/start/quick-start.mdx

```diff
@@ -202,26 +202,6 @@ bun add @module-federation/enhanced
 
 ### 3. Consuming the Producer
 
-> Change the entry to be asynchronous
```

Comment:
It seems that this part is necessary. Why delete it and consider it duplicate?

Reply:
This section was in the page twice:

https://github.com/module-federation/core/pull/2424/files/a1b5018b0dc98bb87119a65a025351ed3ca9f504#diff-7eea34105b6cdc76970c871d5a51543347e429d3d8fa8a755d507a705b151485R248-R266

I removed the first instance rather than the second, as it seemed like a cleaner sequence that way, but either one would work.